app/build.gradle

```diff
@@ -5,12 +5,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
-        applicationId "dev.loveth.roomdemo"
+        applicationId "com.example.roomdemo"
         minSdk 21
-        targetSdk 33
+        targetSdk 34
```

Comment:
target 34 was needed to make the alpha version work

Reply:
I see